CoreScriptsRoot/Modules/Chat.lua

```diff
@@ -1946,18 +1947,14 @@ local function CreateChatWindowWidget(settings)
 				{
 					Name = 'MessageContainer';
 					Size = UDim2.new(1, -SCROLLBAR_THICKNESS - 1, 0, 0);
-					Position = UDim2.new(0, 0, 1, 0);
+					Position = UDim2.new(0, 0, 0, 0);
```

Comment:
Chat is going to start filling in from the top down to the bottom, also requested by Adam